app/build.gradle

```diff
@@ -2,24 +2,21 @@ apply plugin: 'com.android.application'
 
 buildscript {
     repositories {
+        maven { url 'https://jitpack.io' }
         jcenter()
-        maven {
-            url uri("./repo")
-        }
-        maven { url 'https://repo.gradle.org/gradle/repo' }
     }
     dependencies {
         // TODO: haven't updated yet
-        classpath 'rejasupotaro:onesky-gradle-plugin:0.1.0'
+        classpath 'com.github.kazu-matsu:onesky-gradle-plugin:km~rename-packagename-SNAPSHOT'
```

Comment:
temporary code.
After merging and publishing the latest one, I'll change again.